governator-jetty/src/main/java/com/netflix/governator/guice/jetty/JettyModule.java

```diff
@@ -82,6 +84,9 @@ private JettyConfig getConfig() {
  *
  */
 public final class JettyModule extends AbstractModule {
+
+    public static final String PLAINTEXT_CONNECTOR_NAME = "plaintext";
+
     private final static Logger LOG = LoggerFactory.getLogger(JettyModule.class);
```

Comment:
no need for static here; Guice will ensure the singleton nature of this class and logger

Reply:
That was already there, but ok. :)

Reply:
Actually, since the logger is used in the static JettyRunner inner class, it looks like this needs to be static. (And JettyRunner needs to be static since it's a Guice-bound class).

Reply:
sorry didn't notice this was existing code until after I commented.  np to leave as-is.